aten/src/ATen/test/cuda_vectorized_test.cu

```diff
@@ -0,0 +1,124 @@
+#include <gtest/gtest.h>
```

Comment:
Can you also add a test to the main test suite that would test elementwise_kernel with an unrolled tail? I suspect all the regularly run test cases are either too small, or are of size 1024 that would not have a tail.

Reply:
added in `test/test_torch.py`